server/strategies/logout.js

```diff
@@ -0,0 +1,11 @@
+export const logout = (req, res, next) => {
```

Comment:
This isn't exactly a strategy in the passport.js sense - could we move this function to the `utils` directory?

Reply:
Good catch
[89f8b42](https://github.com/AMP-SCZ/dpdash/pull/579/commits/89f8b42c8be681b87e2497243b9c27e31680d918)